test/general/test_policies.pass.cpp

```diff
@@ -77,44 +77,47 @@ main()
     test_policy_instance(oneapi::dpl::execution::make_device_policy<class Kernel_16>());
 
     // device_policy
-    EXPECT_TRUE(device_policy<class Kernel_1>(q).queue() == q, "wrong result for queue()");
-    test_policy_instance(device_policy<class Kernel_21>(q));
-    test_policy_instance(device_policy<class Kernel_22>(sycl::device{TestUtils::default_selector}));
-    test_policy_instance(device_policy<class Kernel_23>(dpcpp_default));
-    test_policy_instance(device_policy<class Kernel_24>(sycl::queue(dpcpp_default))); // conversion to sycl::queue
-    test_policy_instance(device_policy<>{});
+    EXPECT_TRUE(execution::device_policy<class Kernel_1>(q).queue() == q, "wrong result for queue()");
+    test_policy_instance(execution::device_policy<class Kernel_21>(q));
+    test_policy_instance(execution::device_policy<class Kernel_22>(sycl::device{TestUtils::default_selector}));
+    test_policy_instance(execution::device_policy<class Kernel_23>(execution::dpcpp_default));
+    test_policy_instance(execution::device_policy<class Kernel_24>(sycl::queue(execution::dpcpp_default))); // conversion to sycl::queue
+    test_policy_instance(execution::device_policy<>{});
     class Kernel_25;
-    static_assert(std::is_same_v<device_policy<Kernel_25>::kernel_name, Kernel_25>, "wrong result for kernel_name (device_policy)");
+    static_assert(std::is_same_v<execution::device_policy<Kernel_25>::kernel_name, Kernel_25>, "wrong result for kernel_name (device_policy)");
 
 #if ONEDPL_FPGA_DEVICE
-    static_assert(is_execution_policy<fpga_policy</*unroll_factor =*/ 1, class Kernel_0>>::value, "wrong result for is_execution_policy<fpga_policy>");
-    static_assert(is_execution_policy_v<fpga_policy</*unroll_factor =*/ 1, class Kernel_0>>, "wrong result for is_execution_policy_v<fpga_policy>");
+    static_assert(is_execution_policy<execution::fpga_policy</*unroll_factor =*/ 1, class Kernel_0>>::value, "wrong result for is_execution_policy<fpga_policy>");
+    static_assert(is_execution_policy_v<execution::fpga_policy</*unroll_factor =*/ 1, class Kernel_0>>, "wrong result for is_execution_policy_v<fpga_policy>");
     test_policy_instance(dpcpp_fpga);
 
     // make_fpga_policy
     test_policy_instance(TestUtils::make_fpga_policy</*unroll_factor =*/ 1, class Kernel_31>(sycl::queue{TestUtils::default_selector}));
     test_policy_instance(TestUtils::make_fpga_policy</*unroll_factor =*/ 2, class Kernel_32>(sycl::device{TestUtils::default_selector}));
-    test_policy_instance(TestUtils::make_fpga_policy</*unroll_factor =*/ 4, class Kernel_33>(dpcpp_fpga));
+    test_policy_instance(TestUtils::make_fpga_policy</*unroll_factor =*/ 4, class Kernel_33>(execution::dpcpp_fpga));
     // Special case: required to call make_fpga_policy directly from oneapi::dpl::execution namespace
     test_policy_instance(oneapi::dpl::execution::make_fpga_policy</*unroll_factor =*/ 8, class Kernel_34>());
```

Comment:
It probably makes sense to make this consistent with the rest and just use the `execution` namespace.